docs/en_US/Compressor/Pruner.md

```diff
@@ -1,5 +1,4 @@
-Pruner on NNI Compressor
-===
+# Supported Pruning Algorithms on NNI
 
 Index of supported pruning algorithms
```

Comment:
Suggest to organize with fine-grained pruning / structural pruning

Reply:
could you give me a version of such organization?

Reply:
Sure~ let me try it.

Reply:
The `WeightRankFilterPruner`, `ActivationRankFilterPruner` and `GradientRankFilterPruner` have been removed from code. Suggest to flatten the structured pruners.

Reply:
And suggest move lottery ticket to another category , or same category as `AGP`

Reply:
updated, could @chicm-ms make further changes in your pr? thanks!